google/cloud/aiplatform/base.py

```diff
@@ -561,6 +579,123 @@ def _sync_object_with_future_result(
             if value:
                 setattr(self, attribute, value)
 
+    def _construct_sdk_resource_from_gapic(
+        self,
+        gapic_resource: proto.Message,
+        credentials: Optional[auth_credentials.Credentials] = None,
+    ) -> AiPlatformResourceNoun:
+        """Given a GAPIC object, return the SDK representation."""
+        sdk_resource = self._empty_constructor(credentials=credentials)
+        sdk_resource._gca_resource = gapic_resource
+        return sdk_resource
+
+    # TODO(b/144545165) - Improve documentation for list filtering once available
+    @classmethod
+    def list(
+        cls,
+        filter: Optional[str] = None,
+        order_by: Optional[str] = None,
+        page_size: Optional[int] = None,
+        read_mask: Optional[field_mask.FieldMask] = None,
```

Comment:
`page_size` is not respected due to the list comprehension, should we drop it for now?
Also `read_mask` has little use since the user is not getting the `_gca_resource`, think we can hide that as well.

Reply:
I agree that we should remove read_mask for now and we can add it back in later.

Reply:
Dropping both fields and and filed [b/184910159](http://b/184910159) to eventually expose `page_size`